### PR TITLE
Connect node to worker via internal network

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: node/src/main/scala/fluence/node/MasterNode.scala [hunk 1]
```diff
@@ -78,44 +74,42 @@ case class MasterNode[F[_]: ConcurrentEffect: LiftIO](
...
-          for {
-            _ <- IO(logger.info("Running worker `{}`", params)).to[F]
-            newly <- pool.run(params)
-            _ <- IO(logger.info(s"Worker started (newly=$newly) {}", params)).to[F]
-          } yield ()
-      )
-      .compile
-      .drain
+    for {
+      paths <- WorkerConfigWriter.resolveWorkerConfigPaths(app, rootPath)
```
Comment (DieMyst) [inline node/src/main/scala/fluence/node/MasterNode.scala]: log all steps maybe?

File: node/src/main/scala/fluence/node/docker/DockerNetwork.scala [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+// Represents docker network, defined by it's name
+case class DockerNetwork(name: String) extends AnyVal
+
+object DockerNetwork extends slogging.LazyLogging {
+
+  // Run shell command and raise error on non-zero exit code
+  private def run[F[_]: ContextShift](cmd: String)(implicit F: Sync[F]): F[Unit] = {
+    shiftDelay(cmd.!).flatMap {
+      case exit if exit != 0 =>
+        logger.error(s"`$cmd` exited with code: $exit")
```
Comment (DieMyst) [inline node/src/main/scala/fluence/node/docker/DockerNetwork.scala]: will we see the output from errored commands?

File: node/src/main/scala/fluence/node/docker/DockerNetwork.scala [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        F.raiseError[Unit](new Exception(s"`$cmd` exited with code: $exit"))
+      case _ => F.pure(())
+    }
+  }
+
+  // Create docker network as a resource. Network is deleted after resource is used.
```
Comment (DieMyst) [inline node/src/main/scala/fluence/node/docker/DockerNetwork.scala]: write docs in the correct format :smile:

File: node/src/main/scala/fluence/node/workers/tendermint/rpc/TendermintRpc.scala [hunk 4]
```diff
@@ -122,23 +107,27 @@ object TendermintRpc {
   /**
    * Runs a WorkerRpc with F effect, acquiring some resources for it
```
Comment (DieMyst) [inline node/src/main/scala/fluence/node/workers/tendermint/rpc/TendermintRpc.scala]: this doc says nothing

File: node/src/main/scala/fluence/node/workers/tendermint/rpc/TendermintRpc.scala [hunk 5]
```diff
@@ -122,23 +107,27 @@ object TendermintRpc {
...
-   * @param params Worker params to get Worker URI from
    * @param sttpBackend Sttp Backend to be used to make RPC calls
    * @tparam F Concurrent effect
    * @return Worker RPC instance. Note that it should be stopped at some point, and can't be used after it's stopped
    */
-  def apply[F[_]: Concurrent](
-    params: WorkerParams
-  )(implicit sttpBackend: SttpBackend[F, Nothing]): Resource[F, TendermintRpc[F]] =
+  def make[F[_]: Concurrent](
+    hostName: String,
```
Comment (DieMyst) [inline node/src/main/scala/fluence/node/workers/tendermint/rpc/TendermintRpc.scala]: add this arguments to doc

File: node/src/main/scala/fluence/node/MasterNode.scala (hunk 1 above)
Comment (folex) [reply node/src/main/scala/fluence/node/MasterNode.scala]: Why now?

File: node/src/main/scala/fluence/node/docker/DockerNetwork.scala (hunk 2 above)
Comment (folex) [reply node/src/main/scala/fluence/node/docker/DockerNetwork.scala]: Yes

File: node/src/main/scala/fluence/node/workers/tendermint/rpc/TendermintRpc.scala (hunk 4 above)
Comment (folex) [reply node/src/main/scala/fluence/node/workers/tendermint/rpc/TendermintRpc.scala]: this is not code of this PR

File: node/src/main/scala/fluence/node/docker/DockerNetwork.scala [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import fluence.node.docker.DockerIO.shiftDelay
+
+import scala.language.higherKinds
+import scala.sys.process._
+
+/**
+ * Represents docker network, defined by it's name
+ * */
```
Comment (alari) [inline node/src/main/scala/fluence/node/docker/DockerNetwork.scala]: Too many stars

File: node/src/main/scala/fluence/node/workers/DockerWorker.scala [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+  description: String
+) extends Worker[F] {
+
+  // Getter for the last healthcheck
+  val healthReport: F[WorkerHealth] = healthReportRef.get
+}
+
+object DockerWorker extends LazyLogging {
+  // Internal ports
+  val P2P_PORT: Short = 26656
```
Comment (alari) [inline node/src/main/scala/fluence/node/workers/DockerWorker.scala]: Please name the constants scala way (camel case, starting with capital letter).

**Pull request merged**